java/arcs/android/storage/ttl/PeriodicCleanupTask.kt

```diff
@@ -33,7 +33,7 @@ class PeriodicCleanupTask(
         // Use the DatabaseDriverProvider instance of the databaseManager to make sure changes by
         // TTL expiry are propagated to listening Stores.
         val databaseManager = DatabaseDriverProvider.manager
-        databaseManager.removeExpiredEntities().join()
+        databaseManager.removeExpiredEntities()
```

Comment:
this is now swallowing the exceptions, would it better to rethrow?

Reply:
I think you're probably right, since that's what it was originally doing. That seems like the behavior we want for current use cases as well: if removal fails, the result may be that we are violating a policy.

Reply:
Looked again... this is calling directly into a database manager, not via DriverFactory. So there's no `supervisorScope` in this case, and exceptions will still bubble up.

Reply:
Do we want to apply the same supervisorScope + parallel launch pattern to the databasemanager?

Reply:
you are right. What about StorageServiceManager? do we want to propagate OperationFailure via the resultCallback? I think the cleardata coordinator handle failures of delegates by wiping out storage, so it is important that we fail the call if we have not clean all data properly.

Reply:
so I am not sure that returning OperationFailure is a good idea as it may make failures undetected. Why not go back to the previous behavior of simply throwing exceptions?

Reply:
That is already happening in this particular code block.